### PR TITLE
Fix cml-comment pull request sha

### LINKED ISSUE #222: cml-send-comment pull request events is not effective on GH
Pull request UI is not showing commits that happens on PR commits; It's only showing the commits that happens in the branch. Because of that ```cml-send-comment ``` does not apparently work. It does work but is commenting on a commit that is not displayed in the UI

### PR DESCRIPTION
**Pull request by DavidGOrtega**
Pull requests in GH is pointing the sha to the latest commit of the PR branch not even displaying the comment that the user would make in that commit.

closes #222

### CODE REVIEW

**Comment by DavidGOrtega** [comment]
It works now, having two comments one for push and one for pull_request, the "strange" thing is that the UI is merging both as seen below, while the there are two independent comments that can be seen [here](https://github.com/iterative/cml/commit/f47691f4ae6ef3cec4c38c7e019fd8173fa80480)

File: bin/cml-send-comment.test.js [hunk 1]
```diff
@@ -30,9 +30,9 @@ describe('CML e2e', () => {
       "Usage: cml-send-comment.js <path> --head-sha <string>
 
       Options:
-        --version   Show version number                                      [boolean]
-        --head-sha  Commit sha
-        -h          Show help                                                [boolean]"
+        --version     Show version number                                    [boolean]
+        --commit-sha  Commit sha
```
Comment (dmpetrov) [inline bin/cml-send-comment.test.js:34]: This looks like an API change. It is better to keep the old option for backward compatibility with some warning messages and depreciate later (in a few months).

**Review by dmpetrov — APPROVED** [review]
👍 One comment re the compatibility.

File: bin/cml-send-comment.test.js (hunk 1 above)
Comment (DavidGOrtega) [reply bin/cml-send-comment.test.js:34]: sounds good

**Pull request merged**